08-hash-tables/lib/intersecting-tree.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const treeIntersection = (tree1, tree2) => {
+  if (!tree1 || !tree2) { throw new Error('Error: intersection requires two trees'); }
+  let results = [];
+  let traverseCollection = {};
+  let traverseCallBack = (node) => {
+    traverseCollection[node.value] = node.value;
+  };
+  let intersectionCb = (node) => {
```

Comment:
While it's fine to name a parameter in a method definition with `cb`, it is not clean code to name methods with `cb` or `Cb`. `cb` is describing implementation, but method names should be based on context.